linter_exclusions.yml

```diff
@@ -623,6 +623,26 @@ desktopvirtualization workspace update:
     application_group_references:
       rule_exclusions:
       - option_length_too_long
+devcenter admin catalog update:
+  parameters:
+    dev_center_name:
+      rule_exclusions:
+      - no_parameter_defaults_for_update_commands
```

Comment:
Why do you need to use default value for update commands?

Reply:
We would like to use "az configure" for this parameter because it appears in a lot of our commands and it would improve the usability of the commands for the users. You cannot update the dev_center/dev_center_name name for any of these resources - they are child resources of dev_center and is only used to help identify the resource to perform the update operation on (similar to resource group).